src/documents/admin.py

```diff
@@ -138,12 +142,25 @@ class NotesAdmin(GuardedModelAdmin):
     list_display = ("user", "created", "note", "document")
     list_filter = ("created", "user")
     list_display_links = ("created",)
+    raw_id_fields = ("document",)
+    search_fields = ("document__title",)
+
+    def get_queryset(self, request):
+        return (
+            super()
+            .get_queryset(request)
+            .select_related("user", "document__correspondent")
```

Comment:
Similar to the other one, I dont get this (Im just asking, feel free to educate me), since the Notes model doesnt use `document__correspondent`, I think?